packages/expo-notifications/ios/EXNotifications/ServerRegistration/ServerRegistrationModule.swift

```diff
@@ -0,0 +1,166 @@
+//  Copyright © 2024 650 Industries. All rights reserved.
+
+// swiftlint:disable force_unwrapping
```

Comment:
Instead of disabling the rule in the entire file, could you use `// swiftlint:disable:next force_unwrapping` or modify the code so that you don't need force unwrapping? It's generally very bad pattern, even if it's never going to crash.

Reply:
Just my point of view - I agree with silencing the linter line-by-line instead of the whole file, and in this case I'd personally vote for still using force unwraps because the values we use should be present (if not, it's an error), and at the same time we don't want to become "slaves" of swiftlint.

Reply:
The problem is that crashes are pretty bad from the end user and app developers perspective. I kinda agree that for us, library maintainers, it would be different.
Let's imagine a scenario when your app starts crashing for some users, let's say on a newly released device or OS update. Then usually it's better to keep the app functioning with some parts of it being broken rather than entirely crashing the app that frustrates users much more, which converts to bad reviews. In my opinion all native errors should be propagated to JS so that app developers can handle them themselves and decide what's best in their case. You can't easily change the native code of the app, but you can change JS code and do the update – in that case app developers can react to such issues much faster.

Reply:
I've replaced all the unsafe constants, and replaced the unsafe `string.data(with:)` calls to a private function, where I have attempted to make the implementation completely safe.